tissue-clicker/src/Rollers.jsx

```diff
@@ -1,8 +1,29 @@
+import Roller from "./Roller";
+import './App.css';
+
 function Rollers() {
+    const rollerData = [
+        { name: 'Basic Roller', cost: 100, increment: 1, interval: 2000 },
+
+    ];
+
     return (
-        <div>
+        <div className="rollers">
             <h1>Rollers</h1>
             <p>Unlockable components to assist in click count.</p>
+            <div className="roller-list">
+                {rollerData.map(function (roller, index) {
+                    return (
+                        <Roller
+                            key={index}
+                            name={roller.name}
+                            cost={roller.cost}
+                            increment={roller.increment}
+                            interval={roller.interval}
```

Comment:
Thinking of adding a roller.level so players can see how many of each roller they've bought